.github/workflows/linux.yml

```diff
@@ -22,7 +28,7 @@ jobs:
           # FIXME: Edit when KLEE is rebased to latest upstream.
           - {os: ubuntu-20.04, llvm: 11, compiler: gcc, klee: 'no-klee'}
           - {os: ubuntu-20.04, llvm: 12, compiler: gcc, klee: 'no-klee'}
-          - {os: ubuntu-20.04, llvm: 13, compiler: gcc, klee: 'no-klee'}
+          - {os: ubuntu-22.04, llvm: 13, compiler: gcc, klee: 'no-klee'}
```

Comment:
Should not we add new line instead of changing it?

Reply:
No, as I prefer to use official Ubuntu packages when possible^1. LLVM 13 and newer are not packaged for Ubuntu 20.04 so we had to use packages from https://apt.llvm.org/ instead.

^1 LLVM 7 is an exception as the official Ubuntu package is broken since it was never updated to LLVM 7.1. Details are here: https://lists.llvm.org/pipermail/llvm-dev/2018-September/126564.html

Reply:
OK